3.2/s2i/bin/assemble

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+shopt -s dotglob
+echo "---> Installing application source ..."
+ls -A /tmp/a/* &>/dev/null && mv /tmp/src/* ./
```

Comment:
I'm not sure whether I'm missing something or it is a typo -- shouldn't `ls -A /tmp/a/` be rather `ls -A /tmp/src/`?